nilearn/decoding/tests/test_decoder.py

```diff
@@ -979,7 +980,7 @@ def test_decoder_multiclass_error_incorrect_cv(multiclass_data):
 
 def test_decoder_multiclass_warnings(multiclass_data, rng):
     X, y, _ = multiclass_data
-    groups = rng.binomial(2, 0.3, size=len(y))
```

Comment:
@Remi-Gau in some test cases the random numbers generated differ enough between the 2 strategies leading to a failure and so they need a different seed. So in cases where I used the function with an explicit seed like this is because the fixture's seed of 42 fails the test

Reply:
ok but then we can remove rng from the list of arguments, no?
I may need more coffee but I don't see it being called anywhere else in the test.

Reply:
Looks like I need more coffee haha